screens/Menu/CourseSchedule.js

```diff
@@ -23,53 +24,143 @@ import PropTypes from "prop-types";
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 function CourseSchedule(props) {
+    const [accessToken, setAccessToken] = React.useState(null);
+    const [selectedCalendarId, setSelectedCalendarId] = React.useState(null);
+    const [calendarEventsList, setCalendarEventsList] = React.useState(null);
+    const [switchVal, setSwitchVal] = React.useState("false");
+    const [refresh, setRefresh] = React.useState(false);
+
+    //TODO Show only events after current date THIS FORMAT 2020-04-18
+    let currentDate = new Date();
+
+    const getCalendarId = async () => {
+        let calendarId = await AsyncStorage.getItem("calendarId");
+        setSelectedCalendarId(calendarId);
+    };
+
+    const getAccessToken = async () => {
+        let AccessToken = await AsyncStorage.getItem("accessToken");
+        setAccessToken(AccessToken);
+    };
 
-     /**
+    const getSwitchValue = async () => {
+        let switchValue = await AsyncStorage.getItem('switchVal');
+        setSwitchVal(switchValue);
+    };
+
+    const getCalendarEvents = async () => {
+        let calendarEvents = await fetch('https://www.googleapis.com/calendar/v3/calendars/' + selectedCalendarId + '/events', {
+            headers: { Authorization: `Bearer ${accessToken}` }
+        });
+        let resp = await calendarEvents.json();
+        let filteredList = getFilteredGoogleCalendarEvents(resp);
+        let result = filteredList.filter(element => element.summary)
+        setCalendarEventsList(result);
+        setRefresh(false);
+    }
```

Comment:
Modify this method by the following: 
```
  const getCalendarEvents = async () => {
        let calendarEvents = await fetch('https://www.googleapis.com/calendar/v3/calendars/' + selectedCalendarId + '/events', {
            headers: { Authorization: `Bearer ${accessToken}` }
        });
        let resp = await calendarEvents.json();
        if (resp.items.length > 0) {
        let filteredList = getFilteredGoogleCalendarEvents(resp);   
        setCalendarEventsList(filteredList);
        setRefresh(false);
        }
        else {
            setCalendarEventsList({NoEvent: true})
        }
    }
```
**Fixing Issue: ** Error Handling for no events case.

Reply:
FIXED!